docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md

```diff
@@ -0,0 +1,351 @@
+
+```sh
+--role=roles/compute.admin
+```
+
+To grant the required access, you must do the following:
+
+- Create a service account in GCP.
+- [Grant the required roles](https://cloud.google.com/iam/docs/grant-role-console) to the account.
+- Obtain a file containing a JSON that describes the service account credentials.
+
+Retain the service account JSON file. You will need to provide this file later to YBA.
+
+  </div>
+
+  <div id="azure" class="tab-pane fade" role="tabpanel" aria-labelledby="azure-tab">
+
+## Azure
+
+The following permissions are required for the Azure resource group where you will deploy.
+
+```sh
+Network Contributor
+Virtual Machine Contributor 
+```
+
+To grant the required access, you can do one of the following:
+
+- [Register an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) in the Azure portal so the Microsoft identity platform can provide authentication and authorization services for your application. Registering your application establishes a trust relationship between your application and the Microsoft identity platform.
+
+- [Assign a managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/qs-configure-portal-windows-vm) to the Azure VM hosting YugabyteDB Anywhere.
+
+For information on assigning roles to applications, see [Assign a role to an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application); and assigning roles for managed identities, see [Assign Azure roles using the Azure portal](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition) in the Microsoft Azure documentation.
+
+If are registering an application, record the following information about your service account. You will need to provide this information later to YBA.
+
+| | |
+| :--- | :--- |
+| Client ID: | |
+| Client Secret: | |
+| Resource Group: | |
+| Subscription ID: | |
+| Tenant ID: | |
+
+  </div>
+
+  <div id="k8s" class="tab-pane fade" role="tabpanel" aria-labelledby="k8s-tab">
+
+## Kubernetes
+
+As a prerequisite, you need a service account in the Kubernetes cluster and to generate a corresponding KUBECONFIG file.
+
+Perform the following:
+
+- [Create a `yugabyte-platform-universe-management` service account](#create-a-service-account).
+- [Create a `kubeconfig` file](#create-a-kubeconfig-file) of the service account you created to configure access to the Kubernetes cluster.
+
+Do this for each Kubernetes cluster if you are doing a multi-cluster setup.
+
+If YBA is itself installed on Kubernetes, you can use its existing service account to discover details about the Kubernetes cluster and auto-fill the provider configuration. You can then modify these settings to further customize the provider. See [Create a provider](../../../configure-yugabyte-platform/kubernetes/).
```

Comment:
I thought there is no "If YBA is itself installed on Kubernetes" - YBA must be installed on Kubernetes to deploy the YBA universe nodes as well. Right?